reverse.js

```diff
@@ -20,12 +20,20 @@
    * and/or shortcut commands.
    */
   function normalizePath(d) {
+    // regular expression for detecting and replacing multiple decimal points
+    // in a continuous string
+    var regexp = /(\d*\.\d+)((\.\d*)+)/g;
```

Comment:
note that scientific notation is legal, so we'll need an optional `(e-?\d+)?` before that second decimal point to catch things like `"m0 0 L 100 0 L 1.0e2.1e3 L 0 100 z"`

Reply:
Good point, didn't think about that. I fixed it and added a test case.